Look for the X11 unix socket in $PREFIX/tmp/.X11-unix instead of /tmp/.X11-unix,
which does not exist (and is not writable) on Android.

--- a/vendor/x11rb-protocol/src/parse_display/connect_instruction.rs
+++ b/vendor/x11rb-protocol/src/parse_display/connect_instruction.rs
@@ -39,7 +39,7 @@
         targets.push(ConnectAddress::Hostname(host, TCP_PORT_BASE + display));
     } else {
         if protocol.is_none() || protocol.as_deref() == Some("unix") {
-            let file_name = format!("/tmp/.X11-unix/X{display}");
+            let file_name = format!("@TERMUX_PREFIX@/tmp/.X11-unix/X{display}");
             targets.push(ConnectAddress::Socket(file_name));
         }
 
